assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/block-settings/index.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import {
+	InspectorControls,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import type { BlockAttributes } from '@wordpress/blocks';
+import { Icon } from '@wordpress/icons';
+import {
+	thumbnailsPositionLeft,
+	thumbnailsPositionBottom,
+	thumbnailsPositionRight,
+} from '@woocommerce/icons';
+import { useDispatch } from '@wordpress/data';
+import {
+	PanelBody,
+	RangeControl,
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore - Ignoring because `__experimentalToggleGroupControlOption` is not yet in the type definitions.
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore - Ignoring because `__experimentalToggleGroupControl` is not yet in the type definitions.
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControl as ToggleGroupControl,
+} from '@wordpress/components';
+
+interface ContextProps {
+	clientId: string;
+	thumbnailsPosition: string;
```

Comment:
I suggest to create an enum:

```
const enum ThumbnailsPosition = {
   "off" = "off",
   ...
}
```

And refactor this:
```suggestion
	thumbnailsPosition: ThumbnailsPosition;
```

Reply:
Just a small correction on this, when defining an enum, the `const` and `=` sign are not needed, we can declare it like this:
```
enum ThumbnailsPosition { 
  OFF = 'off', 
  ... 
}
```